Code/GraphMol/MolDraw2D/MolDraw2DUtils.cpp

```diff
@@ -267,10 +267,10 @@ void contourAndDrawGaussians(MolDraw2D &drawer,
       maxP.y = std::max(loc.y, maxP.y);
     }
     Point2D dims = maxP - minP;
-    minP.x -= drawer.drawOptions().padding * dims.x;
-    minP.y -= drawer.drawOptions().padding * dims.y;
-    maxP.x += drawer.drawOptions().padding * dims.x;
-    maxP.y += drawer.drawOptions().padding * dims.y;
+    minP.x -= fabs(drawer.drawOptions().padding) * dims.x;
```

Comment:
Are these changes connected to the bug?
It's not clear to me that they are correct. It seems to me that if the padding is negative (for some reason), it should have the corresponding effect.

Reply:
They are not connected. We can leave the padding as is. I was tempted to put a `PRECONDITION` that padding should be >=0 as I don't think a negative padding makes sense, then I thought of a less intrusive way to obtain the same effect. Happy to remove the `fabs()` if you believe they are confusing, and leave the user free to set a negative padding.

Reply:
if for some reason they *want* to have the bounding box be too tight, it doesn't seem like we need to restrict that.

Reply:
@greglandrum done.